src/ClusterBootstrap/deploy.py

```diff
@@ -2452,11 +2457,20 @@ def run_kube(prog, commands):
                                                                                                                              config["k8sAPIport"], "./deploy/ssl/ca/ca.pem", "./deploy/ssl/kubelet/apiserver-key.pem", "./deploy/ssl/kubelet/apiserver.pem", one_command))
     if verbose:
         print(kube_command)
-    os.system(kube_command)
+    if need_output:
+        # we may want to dump command to another file instead of args.output, when we don't want to mix k8s commands with others
+        print("output specified")
+        output = utils.execute_or_dump_locally(kube_command, verbose, False, '')
+        if not args.verbose:
```

Comment:
`args.verbose` -> `verbose`? Why do we want to print it when not verbose? We should print it when verbose as well right?

Reply:
when verbose, it would be printed at line 2463.
if not verbose, this is for kubectl . remember that we decide to output anyway when it's kubectl? so devs don't need to specify -v